### PR TITLE
Enable TSA in AIWin

### PR DESCRIPTION
**Pull request by AhmedAbdoOrtiga**
#### Describe the change
Enable TSA in AIWin as recommended by the SDL process.

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: build/signedbuild.yml [hunk 1]
```diff
@@ -165,6 +168,29 @@ jobs:
...
+      projectNameMSENG: AzureDevOps
+      areaPath: 'AzureDevOps\VSTS\enSPIRe\Keros\Desktop'
+      iterationPath: 'AzureDevOps\OneVS'
+      uploadFxCop: true
+      uploadBinSkim: true
+      uploadCredScan: true
+      uploadAPIScan: false
+      uploadFortifySCA: false
+      uploadModernCop: false
+      uploadPoliCheck: false
```
Comment (RobGallo) [inline build/signedbuild.yml]: Aren't we required to run polycheck?

**Comment by RobGallo** [comment]
Is there a way we can check to make sure this is running as expected?

File: build/signedbuild.yml [hunk 2]
```diff
@@ -279,6 +306,29 @@ jobs:
     displayName: 'Perform Cleanup Tasks'
     condition: succeededOrFailed()
 
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-uploadtotsa.TSAUpload@1
```
Comment (DaveTryon) [inline build/signedbuild.yml]: Should the the upload be before the cleanup task?

File: build/signedbuild.yml [hunk 3]
```diff
@@ -362,4 +414,27 @@ jobs:
       
   - task: ms-vseng.MicroBuildTasks.521a94ea-9e68-468a-8167-6dcf361ea776.MicroBuildCleanup@1
     displayName: 'Perform Cleanup Tasks'
-    condition: succeededOrFailed()
\ No newline at end of file
+    condition: succeededOrFailed()
+
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-uploadtotsa.TSAUpload@1
```
Comment (DaveTryon) [inline build/signedbuild.yml]: Again, should the upload be _before_ the cleanup task?

File: build/signedbuild.yml (hunk 1 above)
Comment (DaveTryon) [reply build/signedbuild.yml]: We definitely need to be running policheck, although I don't know how we specify it via YML. Might be worth checking with the web team to see how they enabled it

**Comment by jalkire** [comment]
Reopening per conversation with @DaveTryon and @RobGallo

**Comment by jalkire** [comment]
Never mind.

**Pull request closed without merging**